### DISCUSSION TITLE
A Simple CSS Trick to Deal with Horizontal Scrollbars in Code Blocks | /en/2023/08/css-scrollbar/

### DISCUSSION THREAD

**Discussion by giscus** [rbind/yihui#1585] [Comments]
The reading experience with horizontal scrollbars is often very bad. For most reading devices, it&rsquo;s easy and natural to scroll vertically, but not so horizontally. It&rsquo;s inconvenient and &hellip;

https://yihui.org/en/2023/08/css-scrollbar/

**Comment by crazypeace** [comment]
Thanks a lot! I just copy and steal. :)
Evey don't think about it more than 3 seconds. LOL
